include/mbedtls/check_config.h

```diff
@@ -629,6 +629,12 @@
         MBEDTLS_ECDSA_C requires MBEDTLS_PK_WRITE_C to be defined."
 #endif
 
+#if defined(MBEDTLS_RSA_C) && defined(MBEDTLS_PKCS1_V15) && \
+    !defined(MBEDTLS_PK_WRITE_C) && defined(MBEDTLS_USE_PSA_CRYPTO)
```

Comment:
I think in this case it should be `MBEDTLS_PSA_CRYPTO_C`, not `MBEDTLS_USE_PSA_CRYPTO`: the failures were in `test_suite_psa_crypto`, which depends on `PSA_CRYPTO_C`, not `USE_PSA_CRYPTO`.

(If we want to be precise we can fix that in `config-mini-tls_1.1.h` as well, though in that particular case in practice those two configs are either both enabled or both disabled, so the difference is less important.)

Reply:
Fixed :)